add some info (#1)
ia-toki/apio-2020-web: rules.md

key: rules
---

# Online Contest Format
* The APIO 2020 online contest is a 5 hour contest with flexible start time within determined window.
* The contest will be held between Friday, 8 May 2020 (UTC+0 06.00) and Sunday, 10 May 2020 (UTC+0 06.00).
* Each country must run one or more official contest sites.

> * All participants must take the contest at an official contest site

Reviewer: ```suggestion
* All participants must take the contest at an official contest site.
```

> * Each contest site starts the contest at the time of its own choice within the window. Different sites from within the same country may also start the contest at different time.

Reviewer: ```suggestion
* Each contest site starts the contest at the time of its own choice within the window. Different sites from within the same country may also start the contest at different times.
```

* Each contest site must be fully supervised in order to guarantee that the contest rules be observed by all the students.

> * Each country may consist of up to 60 students. However, only the top six students will form the official team, according to the contest results.

Reviewer: ```suggestion
* Each country may consist of up to 60 students. However, only the top 6 students will form the official team, according to the contest results.
```

> * Students are eligible for this contest if and only if they are eligible for the IOI 2019.

Reviewer: ```suggestion
* Students are eligible for this contest if and only if they are eligible for the IOI 2020.
```

# Onsite Contest Format

> * The APIO 2020 onsite have the same rule as the online contest format, with several differences.

Reviewer: ```suggestion
* The APIO 2020 onsite contest has the same rule as the online contest format, with several differences.
```

> * There will be only 10 that are invited and 6 participants for each countries. The committee will accept the confirmation of the country that want to do it onsite in First Come First Serve base.

Reviewer: ```suggestion
* There will be only 10 countries that are invited and 6 participants for each countries. The committee will accept the confirmation of the countries that want to do it onsite in First Come - First Serve basis.
```

> * The participants of each contry invited will take the contest at Jakarta, Indonesia.

Reviewer: ```suggestion
* The participants of each invited country will take the contest in Jakarta, Indonesia.
```

> * The contest will start at Sunday, 10 May 2020 (UTC+7 08.00) for 5 hours.

Reviewer: ```suggestion
* The contest will be held on Sunday, 10 May 2020 (UTC+7 08.00) for 5 hours.
```

> * Beside the contest day, there will be a time for excursion and visiting popular place in Jakarta.

Reviewer: ```suggestion
* Beside the contest day, there will be a time for excursion and visiting popular places in Jakarta.
```

> * There's no different between the onsite participants and the online participants. The winner will be decided equally as if the onsite participants are the online participants.

Reviewer: ```suggestion
* There are no differences between the onsite participants and the online participants. The winners will be decided equally as if the onsite participants are the online participants.
```

> * If the onsite participants win, there will be awarded physical medals in closing ceremony / awarding night.

Reviewer: ```suggestion
* If onsite participants win medals, they will be awarded physical medals in closing ceremony / awarding night.
```
